board/opentrons/ot2/rootfs-overlay/etc/nginx/nginx.conf

```diff
@@ -18,7 +18,7 @@ http {
 
         client_body_in_file_only off;
         client_body_buffer_size  128k;
-        client_max_body_size     2M;
+        client_max_body_size     50M;
```

Comment:
Can / should we scope this change specifically to the `POST /protocols` endpoint? I think `2M` is a sensible limit for every endpoint _except_ protocol upload